bigquery_etl/glam/models.py

```diff
@@ -130,6 +130,7 @@ def probe_counts(**kwargs):
 
     return dict(
         attributes=",".join(attributes),
+        attributes_no_os=",".join([attr for attr in attributes if attr != "os"]),
```

Comment:
To be used in `probe_counts_v1.sql`

Reply:
Might be a little cleaner to pass in the array and put this logic in the template but not worth changing.